index.js

```diff
@@ -15,3 +15,70 @@ function total() {
 
   return t
 }
+
+function getCart(){
+  return cart
+}
+
+function addToCart(item){
+var price = Math.floor(Math.random()*99)
+cart.push(new Object({[item]:price}))
+console.log(`${item} has been added to your cart.`)
+return cart
+
+}
+
+function viewCart (){
+if(cart.length === 0){
+  console.log(`Your shopping cart is empty.`)
+}
+else{
+      var myString = "In your cart, you have "
+
+        for(var i = 0, l = cart.length; i < l; i++){
+            for(var list in cart[i]){
+            myString+=`${list} at $${cart[i][list]}`
+                if(i!==cart.length-1){
+                  myString+=", "
```

Comment:
also, where does the last "and" come in, since we are doing something similar to this: "In your cart, you have bananas at $17, pancake batter at $5, and eggs at $49."?